src/module.c

```diff
@@ -900,6 +905,31 @@ long long RM_Milliseconds(void) {
     return mstime();
 }
 
+/* Mark a point in time that will be used as the start time
+ * to calculate the elapsed execution time when
+ * RM_MeasureTimeEnd() is called.
+ * Within the same command, you can call multiple times
+ * RM_MeasureTimeStart() and RM_MeasureTimeEnd() to accummulate 
+ * indepedent time intervals to the background duration.
+ * This method always return REDISMODULE_OK. */
+int RM_MeasureTimeStart(RedisModuleBlockedClient *bc) {
```

Comment:
I think we need a better name for these APIs to reflect the fact that they're only needed for background processing on blocked clients, maybe something like `RM_BlockedClientMeasureTimeStart`
And also refer to this from `RM_BlockClient`

Reply:
@oranagra fixed in the latest commit. can you check it?